src/app/__tests__/app.component.spec.ts

```diff
@@ -33,6 +33,8 @@ import { DateTimeProviderMock } from '@providers/date-time/__mocks__/date-time.m
 
 import { Subscription } from 'rxjs';
 import { SideMenuClosed, SideMenuItemSelected, SideMenuOpened } from '@pages/dashboard/dashboard.actions';
+import { AccessibilityService } from '@providers/accessibility/accessibility.service';
+import { AccessibilityServiceMock } from '@app/__mocks__/accessibility.service.mock';
```

Comment:
Can we move this mock into the same folder as the service, others would typicall follow this structure

```
/provider/
   /__tests__
   /__mocks__
provider.ts   
```